### PR TITLE
fix(setup): Validate minimum number of characters for DigitalTwins name

### PR DESCRIPTION
**Pull request by azabbasi**

### CODE REVIEW

File: sdk/digitaltwins/Azure.DigitalTwins.Core/tests/prerequisites/setup.ps1 [hunk 1]
```diff
@@ -52,11 +53,6 @@ Connect-AzureSubscription
 
 $Region = $Region.Replace(' ', '')
 
-if (-not $DigitalTwinName)
```
Comment (abhipsaMisra) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/tests/prerequisites/setup.ps1:55]: is this check required? `DigitalTwinName` being mandatory now, an error will already have been thrown if it isn't set

Comment (azabbasi) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/tests/prerequisites/setup.ps1:55]: that's why I removed it no?

Comment (abhipsaMisra) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/tests/prerequisites/setup.ps1:55]: oh yes you did, my bad :dizzy_face:
I need to go back to sleep.

**Review by abhipsaMisra — APPROVED** [review]

**Review by timtay-microsoft — APPROVED** [review]

**Review by drwill-ms — APPROVED** [review]
This is good. Of course, I ran into this running New-TestResources.ps1, which won't have this parameter validation.

Can we add that validation to the ARM template so validation happens in both paths?

**Pull request merged**